client/analytics/report/categories/table.js

```diff
@@ -0,0 +1,175 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __, _n } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import { withSelect } from '@wordpress/data';
+import { get, map, orderBy } from 'lodash';
+
+/**
+ * WooCommerce dependencies
+ */
+import { Link, TableCard } from '@woocommerce/components';
+import { formatCurrency, getCurrencyFormatDecimal } from '@woocommerce/currency';
+import { onQueryChange } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import ReportError from 'analytics/components/report-error';
+import { getReportChartData, getReportTableData } from 'store/reports/utils';
+
+class CategoriesReportTable extends Component {
+	getHeadersContent() {
+		return [
+			{
+				label: __( 'Category', 'wc-admin' ),
+				key: 'category',
+				required: true,
+				isLeftAligned: true,
+				isSortable: true,
+			},
+			{
+				label: __( 'Items sold', 'wc-admin' ),
+				key: 'items_sold',
+				required: true,
+				defaultSort: true,
+				isSortable: true,
+				isNumeric: true,
+			},
+			{
+				label: __( 'G. Revenue', 'wc-admin' ),
+				key: 'gross_revenue',
+				isSortable: true,
+				isNumeric: true,
+			},
+			{
+				label: __( 'Products', 'wc-admin' ),
+				key: 'products_count',
+				isSortable: true,
+				isNumeric: true,
+			},
+			{
+				label: __( 'Orders', 'wc-admin' ),
+				key: 'orders_count',
+				isSortable: true,
+				isNumeric: true,
+			},
+		];
+	}
+
+	getRowsContent( categories ) {
+		return map( categories, category => {
+			const { category_id, items_sold, gross_revenue, products_count, orders_count } = category;
+
+			// @TODO it should link to the Products report filtered by category
+			const productsLink = (
+				<Link
+					href={ '/analytics/orders?filter=advanced&code_includes=' + category_id }
+					type="wc-admin"
+				>
+					{ products_count }
+				</Link>
+			);
+
+			return [
+				// @TODO it should be the category name, not the category ID
+				{
+					display: category_id,
+					value: category_id,
+				},
+				{
+					display: items_sold,
+					value: items_sold,
+				},
+				{
+					display: formatCurrency( gross_revenue ),
+					value: getCurrencyFormatDecimal( gross_revenue ),
+				},
+				{
+					display: productsLink,
+					value: products_count,
+				},
+				{
+					display: orders_count,
+					value: orders_count,
+				},
+			];
+		} );
+	}
+
+	getSummary( totals ) {
+		if ( ! totals ) {
+			return [];
+		}
+		return [
+			{
+				label: _n( 'category', 'categories', totals.categories_count, 'wc-admin' ),
+				value: totals.categories_count,
```

Comment:
`numberFormat()` again on all of these non-currency items would be good 👍

Reply:
Sure!